### PR TITLE
update Bars example to demo dataframe constructor

### PR DESCRIPTION
**Pull request by kcpevey**
Updates the Bars example to demo dataframe constructor

I've struggled with creating Bars plots in the past. For this example, it turns out it was really easy to switch to dataframes. I thought it would be a better upgrade, but I'll submit it anyway. 

I'm not sure what aspect I struggled with before (maybe I missed the aggregate step?). If I can get a better reproducible example, I'll create a new PR.

### CODE REVIEW

File: examples/reference/elements/bokeh/Bars.ipynb [hunk 1]
```diff
@@ -100,16 +111,28 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "bars = hv.Bars(df, kdims=['Pets', 'Gender']).aggregate(function=np.sum)\n",
     "\n",
-    "bars.opts(width=500)"
+    "bars.opts(width=800)"
```
Comment (jbednar) [inline examples/reference/elements/bokeh/Bars.ipynb:128]: 800 may be too wide for our docs.

File: examples/reference/elements/bokeh/Bars.ipynb [hunk 2]
```diff
@@ -100,16 +111,28 @@
     "\n",
     "pets_sample = np.random.choice(pets, samples)\n",
     "gender_sample = np.random.choice(genders, samples)\n",
+    "count = np.random.randint(1, 5, size=samples)\n",
     "\n",
-    "bars = hv.Bars((pets_sample, gender_sample, np.ones(samples)), ['Pets', 'Gender']).aggregate(function=np.sum)\n",
+    "df = pd.DataFrame({'Pets': pets_sample, 'Gender': gender_sample, 'Count': count})\n",
+    "df.head(2)"
```
Comment (jbednar) [inline examples/reference/elements/bokeh/Bars.ipynb:117]: This example looks clearer to me than the old one, thanks!

File: examples/reference/elements/bokeh/Bars.ipynb [hunk 3]
```diff
@@ -158,16 +181,29 @@
...
    "name": "python",
-   "pygments_lexer": "ipython3"
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.8.5"
   }
  },
  "nbformat": 4,
- "nbformat_minor": 2
+ "nbformat_minor": 4
```
Comment (jbednar) [inline examples/reference/elements/bokeh/Bars.ipynb:208]: Please strip out the metadata, using something like `strip_kernel`:

```
#!/bin/bash
cp "$1" /tmp/
/usr/local/bin/jq --indent 1 \
    '
    (.cells[] | select(has("outputs")) | .outputs) = []
    | (.cells[] | select(has("execution_count")) | .execution_count) = null
    | .metadata = {"language_info": {"name":"python", "pygments_lexer": "ipython3"}}
    | .cells[].metadata = {}
    ' "$1" > /tmp/stripped.ipynb
cat /tmp/stripped.ipynb > "$1"
```

File: examples/reference/elements/bokeh/Bars.ipynb (hunk 1 above)
Comment (kcpevey) [reply examples/reference/elements/bokeh/Bars.ipynb:128]: didn't mean to change that, I'll fix.

File: examples/reference/elements/bokeh/Bars.ipynb (hunk 3 above)
Comment (kcpevey) [reply examples/reference/elements/bokeh/Bars.ipynb]: Your snippet doesn't actually address the `nbformat`. Should both of those be removed completely?

Comment (jbednar) [reply examples/reference/elements/bokeh/Bars.ipynb]: That's fine to leave; it needs to have some value, and whether that's 4.2 or 4.4 shouldn't matter.

**Comment by philippjfr** [comment]
Thanks @kcpevey!

**Pull request merged**